QueueManagerTest/QueueManagerTests.cs

```diff
@@ -112,5 +138,12 @@ public void TestQueueHasBeenModifiedAfterCopyingList()
             _queueManager.AddMovieToQueue(new List<string> { { "movie2" } });
             Assert.IsTrue(_queueManager.QueueHasBeenModified());
         }
+        [TearDown]
+        public void TearDown()
+        {
+            GC.Collect();
+            GC.WaitForPendingFinalizers();
+            Directory.Delete(_pathString,true);
```

Comment:
weird.  I thought visual studio would have put it there for me.  It does sometimes.